compyle/array.py

```diff
@@ -455,6 +459,7 @@ def get_allocator(queue):
     return allocator
 
 
+@profile_method
```

Comment:
Seems that most times the public API only uses this so it seems that a shorter name would be better, perhaps just `profile` is better?